src/components/logoutlink.test.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import TestUtils from 'react-addons-test-utils';
+import LogoutLink from './LogoutLink';
+
+describe('NavItem', function() {
+	beforeEach(function() {
+		this.logoutLink = TestUtils.renderIntoDocument(<LogoutLink to='/' />);
+		this.linkEl = ReactDOM.findDOMNode(this.logoutLink);
+	});
+
+	it('exists', function() {
+		expect(this.linkEl).not.toBeNull();
+	});
+
+	it('creates an HTML a element', function() {
+		expect(this.linkEl.nodeName).toBe('A');
+	});
+
+	xit('appends the logout query param', function() {
```

Comment:
without a fully-initialized `<Router>`, the `<Link>` component just renders `<a data-reactroot=""></a>`, although I think you can pass an `href` prop to force it to render a particular href in tests. ([source](https://github.com/ReactTraining/react-router/blob/master/modules/Link.js#L103,L129))

But that's not really useful in this case, so I think it would be better to inspect `this.logoutLink` to see if it renders a `<Link>` with a `to` prop that includes `{query: {logout: true}}` - you technically should avoid testing what `<Link>` actually renders anyway and just treat it as a black box.